<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>postsearch</class>
 <widget class="QDialog" name="postsearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询快递界面</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>461</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_keyboard_postsearch">
    <item row="0" column="0">
     <widget class="QPushButton" name="pb1">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pbDelete">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pb0">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pb7">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pb2">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pb3">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pbReturn">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pb9">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pb8">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pb4">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pb5">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pb6">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="splitter_5_postsearch">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QSplitter" name="splitter_4_postsearch">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QSplitter" name="splitter_3_postsearch">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_2_postsearch">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QCheckBox" name="cbNumber">
       <property name="text">
        <string>快递单号</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leNumber">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_postsearch">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QCheckBox" name="cbTel">
       <property name="text">
        <string>手机号码 </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leTel"/>
     </widget>
    </widget>
    <widget class="QPushButton" name="pbSearch">
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tvResuls"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
